src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NuGetInstaller.cs

```diff
@@ -61,7 +62,7 @@ public async Task<IReadOnlyList<CheckUpdateResult>> GetLatestVersionAsync(IEnume
                         }
                         try
                         {
-                            return _updateChecker.GetLatestVersionAsync(nugetSource);
+                            return _updateChecker.GetLatestVersionAsync(nugetSource, CancellationToken.None);
```

Comment:
FYI: I like `default` keyword instead of `CancellationToken.None`, I'm not saying this should be changed, just mentioning it...